src/pages/landing/stocks/index.tsx

```diff
@@ -93,6 +128,7 @@ const Stocks = () => {
     const handleResizeWindow = () => {
         setMobile(isBrowser() ? window.screen.width <= size.mobileL : false)
     }
+    const { is_eu_country } = React.useContext(DerivStore)
```

Comment:
Had to use is_eu_country because for some reason is_eu was working on my localhost and not on the testlink

Reply:
This is good as deprecated. Please use `is_eu` from `useCountryRule`

Reply:
I used that but it wasn't showing the results for eu on the test link. Will try again I suppose